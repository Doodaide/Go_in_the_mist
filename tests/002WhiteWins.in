who
place A2
place A1
place A3
place B2
place A4
place C3
place E1
place D4
place E2
place E5