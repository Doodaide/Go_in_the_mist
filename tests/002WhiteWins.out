B
White wins!
A2A1A3B2A4C3E1D4E2E5
Thank you for playing!
